scanner/src/main/kotlin/scanners/ScanCodeResultParser.kt

```diff
@@ -128,6 +128,8 @@ private fun getLicenseFindings(result: JsonNode): List<LicenseFinding> {
     files.flatMapTo(licenseFindings) { file ->
         val licenses = file["licenses"]?.asSequence().orEmpty()
         licenses.map {
+            println("XXXXXXXXXXXXXXXXXXXXXxXXX")
```

Comment:
Commit message nit: The "I'll investigate" part does not really fit to the passive wording we use in commit messages and could be reworded.

Reply:
Done.